apps/playground/src/app/components/datetime/date-range-picker/date-range-picker.component.html

```diff
@@ -12,6 +12,7 @@
     <div class="sky-margin-stacked-lg">
       <sky-date-range-picker
         formControlName="lastDonation"
+        hintText="Really long hint text that should wrap and be confined to below the file drop element. We set the text to have a small margin below the drop and link controls, and if there are any errors (shown below) there will be a slight spacing between this element and those errors."
```

Comment:
The text here is talking about file drop. Minor - but probably should tweak it to be about the date range picker. :)